src/multiformats/index.ts

```diff
@@ -0,0 +1,4 @@
+import * as codec from './codecs'
```

Comment:
I don't really see any point to have this `codecs` export, moreover, it just make rigid the serialization interfaces (and also you missed some parameter already).

Reply:
I am just using the same pattern as found in multiformats samples. Do not want to change that unless is a must

Reply:
please just remove it, it does not have any sense to wrap the serialization methods with nothing.